packages/abap-deploy-config-inquirer/src/prompts/conditions.ts

```diff
@@ -225,6 +225,9 @@ export function defaultOrShowSearchPackageQuestion(
     useAutocomplete = false
 ): boolean {
     // Only show the autocomplete prompt when the autocomplete prompt is supported; CLI or YUI specific version
+    if (!useAutocomplete) {
```

Comment:
This is a change in behaviour, did we not support CLI usage without autocomplete previously,. The following logic seems redundant now:
```
return (
        (isCli || useAutocomplete) &&
```

Reply:
Updated.